### PR TITLE
feature: Domain capture PUL-691

### PR DESCRIPTION
**Pull request by prcr**
This is a very early draft of the documentation for the domain capture (joining via email domain) feature.

Live preview:

https://deploy-preview-74--docs-pulse.netlify.app/#3-sharing-access-with-teammates

### :construction: To do

- [x] Include details about "edge cases"
- [x] Update screenshots with final versions

### CODE REVIEW

**Review by claudiacodacy — COMMENTED** [review]
Looking good Paulinho.

Edge cases:
- You can only enable "Joining via email domain" to your own domain (keep this in mind when changing this setting if the "active domain is different from your own)
- "Joining via email domain" is only available for Company domains. Gmail or other free domains will not be able to manipulate this feature (you cannot activate or deactivate)
- "Joining via email domain" can only be active for a single organization. If one domain is already being captured by another org, you won't be able to activate it for yours. If your domain is already being captured and you don't understand why/where, contact support.

There's also the case where someone gets invited to org A but has their domain being captured to org B. (You join both orgs and get redirected to A). But honestly I think we don't need to cover that. :)

GG, thank you paulo!

File: docs/index.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ Currently, Pulse provides a push-based integration with your workflow using one
...
+The most convenient way of ensuring that all the members of your company have access to the Pulse dashboards is by enabling the feature **Joining via email domain** on the **Organization members** page. When this option is enabled, anyone who signs up to Pulse using an email address from your company is automatically added to your Pulse organization.
 
-![Inviting teammates to Pulse](images/inviting-members.png)
+Alternatively, copy the invitation link from the **Organization members** page and share the link with specific people. A convenient way to do this is to share and pin the link on your team's chat. The link is valid for 3 months, but you can refresh it at any moment.
 
-By doing this, the people you invite will have access to all dashboards immediately after signing up without needing to perform any integration.
+![Adding organization members](images/joining-email-domain.png)
 
-The link is valid for 3 months, but you can refresh it at any moment.
+The people who sign up using your company's email addres or that you explicitly invite will have access to all dashboards immediately after signing up without needing to perform any integration.
```
Comment (claudiacodacy) [inline docs/index.md:65]: > The people who sign up using your company's email **address** or that you explicitly invite

I found this a little long, but I understand the reasoning. We could try something like "New members", "Newcomers", or "People who signup via email domain or invitation link"

Just a couple of thoughts. <3 Everything is looking really clear ☀️

**Review by claudiacodacy — COMMENTED** [review]
_

File: docs/index.md (hunk 1 above)
Comment (prcr) [reply docs/index.md:65]: I like this suggestion, sometimes it happens that I'm being very precise but a more conversational style works best. :ok_hand:

**Comment by prcr** [comment]
> Edge cases:

Thank you @claudiacodacy! I included a description of these "edge cases" in a note at the end of the page, so that it wouldn't interrupt the "happy path" flow of instructions for getting started with Pulse. You can see how it looks like in the preview:

https://deploy-preview-74--docs-pulse.netlify.app/#3-sharing-access-with-teammates

> There's also the case where someone gets invited to org A but has their domain being captured to org B. (You join both orgs and get redirected to A). But honestly I think we don't need to cover that. :)

Yes, let's simplify! :raised_hands:

File: docs/index.md [hunk 2]
```diff
@@ -56,10 +56,25 @@ Currently, Pulse provides a push-based integration with your workflow using one
...
-![Inviting teammates to Pulse](images/inviting-members.png)
+Alternatively, copy the invitation link from the **Organization members** page and share the link with specific people. A convenient way to do this is to share and pin the link on your team's chat. The link is valid for 3 months, but you can refresh it at any moment.
 
-By doing this, the people you invite will have access to all dashboards immediately after signing up without needing to perform any integration.
+![Adding organization members](images/joining-email-domain.png)
 
-The link is valid for 3 months, but you can refresh it at any moment.
+Newcomers will have access to all dashboards immediately after signing up without needing to perform any integration.
+
+!!! note "About joining via email domain"
```
Comment (afsmeira) [inline docs/index.md:67]: Not sure if it's relevant to mention that turning off this feature won't remove from the org users that joined when the feature was active.

Comment (prcr) [reply docs/index.md:67]: It's a good point, and I've added it to the list of "further considerations" about the feature.

**Review by afsmeira — APPROVED** [review]

**Pull request merged**